eks/managing_user_access.md

```diff
@@ -0,0 +1,107 @@
+# Managing Access to your Kubernetes cluster
+The following document assumes you have used the terraform scripts found in this repository to create a Kubernetes cluster for your CircleCI server installation. Below we go through our recommended approach to providing and managing user access.
+
+
+## Adding Cluster Administrators
+Cluster admins are first registered via appending their IPs and IAM user details to the appropriate lists in your `terraform.tfvars` file.
+
+example:
+```
+allowed_cidr_blocks = ["user_ip/32","user1_ip/32"]
+k8s_administrators = [
+  {
+      groups   = ["system:masters"]
+      userarn  = "<user ARN>"
+      username = "<username>"
+  },
+  {
+      groups   = ["system:masters"]
+      userarn  = "<user1 ARN>"
+      username = "<user1name>"
+  }
+}
+```
+
+Upon `terrafrom apply`, these fields will whitelist your IPs, and add the list of users will be added to the system:masters group for full access to all resources in the cluster.
+
+### Updating the Cluster admin list
+If you wish to add/remove admin users from your existing cluster, you only need to update the values in your `terraform.tfvars` and run `terrafrom apply`.
+
+## Adding Users with Limited Resource Access
+You may not wish for each user to have complete access of to all cluster resources. To more finely tune user access we will make use kubernetes role based access control ([RBAC]).
```

Comment:
Either "we will make use of" or just "we will use"

Reply:
ah good catch. thanks